### ISSUE TITLE
Working with a chainsaw is unreasonably exhausting

### ISSUE THREAD

**Issue by dom4op** [CleverRaven/Cataclysm-DDA#30481]
# Describe the bug
Using a chainsaw to chop down trees or cut trunks into logs makes your char tired in matter of hours.

# Steps To Reproduce
1. Have a good night's sleep. Wake up feeling well rested.
2. Drink some water until you're hydrated. Grab a chainsaw (I used electric chainsaw, but suspect that a regular fueled one has the same issue)
3. Start felling trees.
4. Your thirst meter will drop down from "hydrated" to "thirsty" in less than an hour. In a few hours you'll get tired. In about 6 hours of work you'll be exhausted.

# Expected behavior
I expect to not get tired so rapidly, because I am using a chainsaw. I get that woodwork is tiresome activity. I get it that my char gets tired after chopping down a few trees with a wood axe. But it seems like the game applies the same effort penalty to a chainsaw as it does to an axe.

Okay, operating a 6,5 kg chainsaw is not the same as sewing clothes, it should probably be slightly more tiresome. But the current state of things is I can work 6-8 hours bashing trees with my hatchet until I get tired OR I can use a chainsaw and get tired in 2 hours.

**Comment by Greevv** [comment]
I mean, as someone who has personally felled trees with a chainsaw, you're gonna be exhausted within 3-4 hours.

**Comment by dom4op** [comment]
Will you also be rapidly dehydrated? I admit I never felled a tree, but somehow it feels that chopping it with an axe should be much more exhausting than using a chainsaw.

**Comment by Rail-Runner** [comment]
Well, chainsaws were probably invented for a reason. If it was easier to use an axe, I don't think chainsaws would see any practical use.

**Comment by ifreund** [comment]
I think the bug here is that axes are too easy to use, not that chainsaws are too straining.

**Comment by dom4op** [comment]
Okay, tweaking axes and making them more straining will partially solve the issue. But the sheer amount of water you need to work with a chainsaw is still insane. We're talking about several L of water in matter of hours.

**Comment by CSHague** [comment]
Yeah, I've noticed this issue myself recently. The actual act of felling the tree isn't the hard part (I've done it myself) The hard part is de-limbing and chopping up the logs and moving them. Using a logging chainsaw makes the task laughably easy compared to chopping with an axe. Currently in-game you can cut down several trees with a chainsaw and then suddenly need an entire night's sleep. That's not right. Not at all.

**Comment by wapcaplet** [comment]
Still reproducible in 0.E-6177-g 9f4a1b841d despite the new weariness system. After only 2h20m of chopping down trees with an electric chainsaw, I am not just "Moderately Weary" (as would be expected), but also "Tired" (sleepy):

![image](https://user-images.githubusercontent.com/238395/94326383-bf523300-ff60-11ea-940d-414adb666fbc.png)
